src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -261,6 +265,19 @@ public void runReplay(ReplayBean item) {
     }
   }
 
+  public void runRehost(ReplayBean item) {
+    if (item.getReplayFile() != null) {
+      try {
+        runRehostFile(item.getReplayFile());
+      } catch (Exception e) {
```

Comment:
I would be more specific with the caught errors

Reply:
Required test changes because the 'FakeException' doesn't match a more specific catch statement.